pset-functions.js

```diff
@@ -30,7 +30,24 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
-
+// create function called blackJack w/ 2 parameters (playerCardScore, dealerCardScore)
+function blackJack(playerCardScore, dealerCardScore){
+  //if both players are over 21, return 0
+  if(playerCardScore > 21 && dealerCardScore > 21){
+    return 0
+    //if playerCardScore >= 21, && dealerCardScore >= 21 && playerCardScore > dealerCardScore, return higher score
+  } else if(playerCardScore <= 21 && dealerCardScore <= 21){
+    return Math.max(playerCardScore, dealerCardScore)
```

Comment:
Matt, great use of if/else statements, they are cascading in the proper logic flow, and taking advantage of js' built in Math methods.